schema/test.schema

```diff
@@ -8,6 +8,6 @@ type test.full = record{
  t: time #default="uniform(0,10)",
  d: duration #default="uniform(100,200)",
  a: addr #default="uniform(0,2000000)",
- s: subnet #default="uniform(1000,2000)",
+ u: subnet #default="uniform(1000,2000)",
```

Comment:
I feel that testing the feature by introducing a mismatch between the schemas we ship and the hard-coded ones is difficult to argue. Instead, you could the custom schema in the integration tests, no?

Reply:
This is a bugfix. There's a second field above also called `s` that has a different type. This triggered the newly introduced mechanism on every start of the type registry.